### PR TITLE
[frontend] Fix options for tone in message (#1126)

### PR DESCRIPTION
**Pull request by savacano28**
### Proposed changes

*
*

### Related issues

* https://github.com/OpenBAS-Platform/openbas/issues/1126
*

### Checklist

- [x] I consider the submitted work as finished
- [ ] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: openbas-front/src/admin/components/common/form/TextFieldAskAI.tsx [hunk 1]
```diff
@@ -398,13 +397,19 @@ const TextFieldAskAI: FunctionComponent<TextFieldAskAiProps> = ({
...
-                <MenuItem value="strategic">{t('Strategic')}</MenuItem>
+                <MenuItem value="informal">{t('Informal')}</MenuItem>
+                <MenuItem value="formal">{t('Formal')}</MenuItem>
+                <MenuItem value="assertive">{t('Assertive')}</MenuItem>
+                <MenuItem value="sarcastic">{t('Sarcastic')}</MenuItem>
+                <MenuItem value="authoritative">{t('Authoritative')}</MenuItem>
+                <MenuItem value="bitter">{t('Bitter')}</MenuItem>
+                <MenuItem value="critical">{t('Critical')}</MenuItem>
+                <MenuItem value="arrogant">{t('Arrogant')}</MenuItem>
+                <MenuItem value="aggressive">{t('Aggressive')}</MenuItem>
```
Comment (Dimfacion) [inline openbas-front/src/admin/components/common/form/TextFieldAskAI.tsx:412]: Nitpick : Maybe we should set them in alphabetic order ?

Comment (savacano28) [reply openbas-front/src/admin/components/common/form/TextFieldAskAI.tsx:412]: I'll ask the product team

**Comment by savacano28** [comment]
Product told me to keep this order.

**Review by Dimfacion — APPROVED** [review]
Looks good to me then :)

**Pull request merged**